XtermSharp/Pty.cs

```diff
@@ -21,7 +21,7 @@ public class Pty {
 		[DllImport ("/cvs/XtermSharp/helper/libpty.dylib", EntryPoint ="fork_and_exec")]
 		extern static int HeavyFork (string process, string [] args, string [] env, out int master, UnixWindowSize winSize);
 
-		static bool HeavyDuty = true;
+		static bool HeavyDuty = false;
```

Comment:
I dont think we want this, what is the reason for this change?

The reason for the library is that otherwise, this breaks on Catalina.